src/sage/knots/knotinfo.py

```diff
@@ -1714,9 +1720,23 @@ def khovanov_polynomial(self, var1='q', var2='t', base_ring=ZZ, original=False):
 
         .. NOTE ::
 
-            The Khovanov polynomial given in KnotInfo corresponds to the mirror
-            image of the given knot for a `list of 140 exceptions
-            <https://raw.githubusercontent.com/soehms/database_knotinfo/main/hints/list_of_mirrored_khovanov_polynonmial.txt>`__.
+            The data used here were calculated with the program
+            `KhoHo <https://github.com/AShumakovitch/KhoHo>`__. They are no longer
+            visible on the website as of October 30, 2022. Instead, data
+            computed with `KnotJob <https://www.maths.dur.ac.uk/users/dirk.schuetz/knotjob.html>`__
+            are now displayed. The latter program is more accurate in terms of
+            orientation and reflection as it is based on `PD` code.
+
```

Comment:
The markup of PD and KhoHo is not right, I think. Should be double backticks or no backticks

Reply:
> BTW: Do we have a replacement for the documentation badge on the Trac ticket header? That has been very convenient!

Obviously I was to unpationed!